### PR TITLE
Add 2.11.0 changelog

### PR DESCRIPTION
**Pull request by kgyrtkirk**
# Description

Add changelog for Timescaledb 2.11.0 

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: about/release-notes/index.md [hunk 1]
```diff
@@ -17,6 +17,78 @@ GitHub and be notified by email whenever a new release is available. On the
...
+
+## TimescaleDB&nbsp;2.11.0 on 2023-05-12
+
+This release contains new features and bug fixes since the 2.10.3 release.
+We deem it moderate priority for upgrading.
+
+
+### Highlighted features in this release
+
+This release includes these new features:
```
Comment (iroussos) [inline about/release-notes/index.md:29]: Removing duplicate..

```suggestion
```

File: about/release-notes/index.md [hunk 2]
```diff
@@ -17,6 +17,78 @@ GitHub and be notified by email whenever a new release is available. On the
...
+
+This release includes these noteworthy features:
+* Support for DML operations on compressed chunks:
+  * UPDATE/DELETE support
+  * Support for unique constraints on compressed chunks
+  * Support for `ON CONFLICT DO UPDATE`
+  * Support for `ON CONFLICT DO NOTHING`
+* Join support for hierarchical Continuous Aggregates
+
```
Comment (iroussos) [inline about/release-notes/index.md:38]:
```suggestion
* Performance improvements for real-time Hierarchical Continuous Aggregates

```

**Review by iroussos — CHANGES_REQUESTED** [review]
Thank you @kgyrtkirk, other than two small suggestions, looks good to me!

File: about/release-notes/index.md [hunk 3]
```diff
@@ -17,6 +17,78 @@ GitHub and be notified by email whenever a new release is available. On the
...
+This release includes these new features:
+
+This release includes these noteworthy features:
+* Support for DML operations on compressed chunks:
+  * UPDATE/DELETE support
+  * Support for unique constraints on compressed chunks
+  * Support for `ON CONFLICT DO UPDATE`
+  * Support for `ON CONFLICT DO NOTHING`
+* Join support for hierarchical Continuous Aggregates
```
Comment (Loquacity) [inline about/release-notes/index.md:37]:
```suggestion
* `JOIN` support for hierarchical continuous aggregates
```

File: about/release-notes/index.md [hunk 4]
```diff
@@ -17,6 +17,79 @@ GitHub and be notified by email whenever a new release is available. On the
...
+* Performance improvements for real-time Hierarchical Continuous Aggregates
```
Comment (Loquacity) [inline about/release-notes/index.md:38]:
```suggestion
* Performance improvements for real-time hierarchical continuous aggregates
```

File: about/release-notes/index.md [hunk 5]
```diff
@@ -17,6 +17,78 @@ GitHub and be notified by email whenever a new release is available. On the
 click `Watch`, select `Custom` and then check `Releases`.
 </Highlight>
 
+
+## TimescaleDB&nbsp;2.11.0 on 2023-05-12
+
+This release contains new features and bug fixes since the 2.10.3 release.
+We deem it moderate priority for upgrading.
```
Comment (Loquacity) [inline about/release-notes/index.md:24]: There's a highlight block to use for this, in the template.

Comment (kgyrtkirk) [reply about/release-notes/index.md:28]: added it; along with:
```
These release notes are for the release of TimescaleDB&nbsp;2.11.0 on
2023-05-22.
```

which seem redundant - but its there for earlier entries

Comment (Loquacity) [reply about/release-notes/index.md:28]: It's just a sentence to avoid having a title with no content under it. It helps to improve readability.

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
The deploy to docs job failing is expected. This can be merged whenever you're ready 👍

**Comment by kgyrtkirk** [comment]
@iroussos  could you please take another look?

**Review by iroussos — APPROVED** [review]
Thank you @kgyrtkirk @Loquacity, final version looks good to me!

**Comment by kgyrtkirk** [comment]
@Loquacity seems like `Docs dev deploy` is a required check - so the PR can't be merged without passing it;

I've tried to:
* rebuild that job
* rebase the changes to the tip of the latest branch

I think it might be related to my github user - I believe that secret is not because I'm not in some group - could that be the case?

**Comment by Loquacity** [comment]
> @Loquacity seems like `Docs dev deploy` is a required check - so the PR can't be merged without passing it;
> 
> I've tried to:
> 
> * rebuild that job
> * rebase the changes to the tip of the latest branch
> 
> I think it might be related to my github user - I believe that secret is not because I'm not in some group - could that be the case?

No, it's because you're building off a fork. We can't make that test pass, so I'm happy to bypass the branch protections and merge it for you now 👍

**Comment by kgyrtkirk** [comment]
thank you @Loquacity ; I didn't noticed that the fork is the issue - I'll keep it in mind next time!

**Pull request merged**